compiler/mx.compiler/mx_updategraalinopenjdk.py

```diff
@@ -126,8 +126,12 @@ def run_output(args, cwd=None):
         m_src_dir = join('src', m.name)
         mx.log('Checking ' + m_src_dir)
         out = run_output(['hg', 'status', m_src_dir], cwd=jdkrepo)
```

Comment:
Couldn't this be done with a single regex? Something like: `out = re.sub(r'^\?.*\$', '', out, re.MULTILINE)` (probably not 100% correct).

Reply:
Sure, that works. Updated

Reply:
And now it doesn't work...

Let me look into it.

Reply:
Oh, I figured it out. `re.MULTILINE` was being passed as `count` argument be accident.

I'll update again :-/